app/v3/rand/internal/types/request.go

```diff
@@ -26,13 +27,15 @@ func NewRequest(
 	txHash cmn.HexBytes,
 	oracle bool,
 	serviceFeeCap sdk.Coins,
+	serviceContextID cmn.HexBytes,
```

Comment:
add `serviceContextID`  in
`func (r Request) String() string {}`
and 
`(rs Requests) String() string {}`